### DISCUSSION TITLE
Concept for upstream issue tracking for tractusx-edc releases

### DISCUSSION THREAD

**Discussion by matgnt** [eclipse-tractusx/tractusx-edc#1219] [General]
## WHAT

Track upstream EDC issues for upcoming releases

## WHY
To make it clear for users what to expect and what important bugs / feature requests are fixed in which release.

To make sure we don't miss an important upstream issue that is mandatory for a specific tractusx release.

## HOW

Multiple options:
- open issues in this repo for such important upstream issues
- have a separate tracking repo
- ...?

##
Matthias Binzer

**Comment by paullatzelsperger** [comment]
that is not an issue, but a discussion - at best.

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
